package.json

```diff
@@ -123,7 +123,7 @@
     "terra-dev-site": "^5.0.0",
     "terra-disclosure-manager": "^4.9.0",
     "terra-enzyme-intl": "^3.0.0",
-    "terra-toolkit": "^5.0.0",
+    "terra-toolkit": "https://github.com/cerner/terra-toolkit.git#master",
```

Comment:
Can this now be switched back to ^5.3.0?

Reply:
Waiting for release of 5.4.0 as it contains the Terra.hideInputCaret helper.

Reply:
Since  5.4.0 was released yesterday changing the version to 5.4.0 as it contains Terra.hideInputCaret  in 4738c3e